src/index.js

```diff
@@ -0,0 +1,166 @@
+import './index.css';
+
+// Declaring variables
+
+const myListItems = document.querySelector('.to-do-list');
+const toDoItem = document.getElementById('toDoItem');
+
+// Get data from local storage
+
+const getBookStorage = () => {
+  let taskListStorage;
+  if (JSON.parse(localStorage.getItem('taskList')) === null) {
+    taskListStorage = [];
+  } else {
+    taskListStorage = JSON.parse(localStorage.getItem('taskList'));
+  }
+  return taskListStorage;
+};
+
+// Remove task from local storage
+
+const removeFromStorage = (listIndex) => {
+  const tasks = getBookStorage();
+  const newTaskArray = tasks.filter(
+    (task) => task.index !== parseInt(listIndex, 10),
+  );
+  localStorage.setItem('taskList', JSON.stringify(newTaskArray));
```

Comment:
- Please update the index of items in the list after deleting. Take a look at this [StackOverflow](https://stackoverflow.com/questions/41908302/how-to-update-array-object-index-when-i-delete-a-array-object-item-from-array-in) answer to learn how to.